independent-publisher-connectors/IBM Watson Assistant/apiDefinition.swagger.json

```diff
@@ -0,0 +1,332 @@
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "session_id": {
+                  "type": "string",
+                  "description": "session_id"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create a session",
+        "description": "Create a new session. A session is used to send user input to a skill and receive responses.",
+        "operationId": "CreateSession",
+        "parameters": [
+          {
+            "name": "version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2021-11-27",
+            "description": "date of release of Watson Assistant version",
+            "x-ms-summary": "version"
+          }
+        ]
+      }
+    },
+    "/sessions/{session}": {
+      "delete": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
+          }
+        },
+        "summary": "Delete session",
+        "description": "Deletes a session explicitly before it times out.",
+        "operationId": "DeleteSession",
+        "parameters": [
+          {
+            "name": "session",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "session",
+            "x-ms-summary": "session"
+          },
+          {
+            "name": "version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2021-11-27",
+            "description": "date of release of Watson Assistant version",
+            "x-ms-summary": "version"
+          }
+        ]
+      }
+    },
+    "/sessions/{session}/message": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "output": {
+                  "type": "object",
+                  "properties": {
+                    "intents": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "intent": {
+                            "type": "string",
+                            "description": "intent"
+                          },
+                          "confidence": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "confidence"
+                          }
+                        }
+                      },
+                      "description": "intents"
+                    },
+                    "entities": {
+                      "type": "array",
+                      "items": {},
```

Comment:
@lucastitus is it possible to define the items? maybe the type? Please fix all empty properties in the swagger file.